LSP.md

```diff
@@ -0,0 +1,44 @@
+## Kite-LSP
+
+Kite comes with a language server adapter that may be used to enable Kite
+completions via your favorite LSP client.
+Other Kite features than completions are not well-supported via LSP, so you
+should still install the Kite plugin for Vim.
+
+The Kite-LSP adapter may be run as follows, depending on your operating system:
+```bash
+/Applications/Kite.app/Contents/MacOS/kite-lsp --editor=vim
+~/.local/share/kite/current/kite-lsp --editor=vim
+C:\Program Files\Kite\kite-lsp.exe --editor=vim
+```
+
+If you installed Kite to a non-standard location, you may need to appropriately
+modify the executable path.
+
+### Coc.nvim
+
+We have tested Kite-LSP with the full-featured
+[coc.nvim](https://github.com/neoclide/coc.nvim) completions plugin.
+However, it should likely work with most LSP client plugins.
+
+In order to enable Kite-LSP, add the following to your coc.nvim configuration
+(`:CocConfig`):
+```json
+{
+  "languageserver": {
+    "kite": {
+      "command": "/Applications/Kite.app/Contents/MacOS/kite-lsp --editor=vim",
+      "filetypes": ["python", "go", "javascript"]
+    }
+  }
+}
+```
+
+Add to `"filetypes"` any languages for which you would like to enable Kite
+completions.
+
+You should also disable the Kite plugin's completions for those languages to
+avoid getting duplicate completions:
+```viml
+let g:kite_support_language=[]
```

Comment:
I'm not sure this is quite right, since we only want to disable some features, but not all features.

Reply:
That would indeed disable all features.

Do you want to disable both manual and automatic completions or just automatic ones?  To disable the automatic ones: `let g:kite_auto_complete=0`.  There isn't a way to disable manually invoked completions at the moment, though we could certainly make that possible.